tests/context.py

```
@@ -964,44 +969,44 @@ def _should_get_package_version_WHEN_valid(self):
 
 		"""
 		try:
-			self.assertIsNotNone(multicast.__module__, """Version will be efectivly None.""")
-			self.assertIsNotNone(multicast.__version__, """Version is not valid.""")
+			self.assertIsNotNone(multicast.__module__, "Version will be efectivly None.")
```

Ruff (0.8.2)

Use a regular `assert` instead of unittest-style `assertIsNotNone`

Replace `assertIsNotNone(...)` with `assert ...`

(PT009)